packages/common/src/services/integrations/jwp/JWPCheckoutService.ts

```diff
@@ -46,9 +46,9 @@ export default class JWPCheckoutService extends CheckoutService {
     };
   };
 
-  private formatOffer = (offer: AccessFee): Offer => {
+  private formatOffer = (offer: AccessFee, assetId: string): Offer => {
     const ppvOffers = ['ppv', 'ppv_custom'];
-    const offerId = ppvOffers.includes(offer.access_type.name) ? `C${offer.id}` : `S${offer.id}`;
+    const offerId = ppvOffers.includes(offer.access_type.name) ? `C${assetId}_${offer.id}` : `S${assetId}_${offer.id}`;
```

Comment:
Does it make sense to include assetId in the Offer object and to use it where we need it?